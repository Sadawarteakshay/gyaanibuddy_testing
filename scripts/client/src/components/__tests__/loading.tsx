import React from 'react';

import { render, cleanup } from '../../test-utils';
import Loading from '../loading';
import renderer from 'react-test-renderer';

describe('Loading', () => {
  // automatically unmount and cleanup DOM after the test is finished.
  afterEach(cleanup);

  it('renders without error', () => {
    render(<Loading />);
  });

  it('render snapshot', () => {
    const LoadingElement = <Loading />;

    render(LoadingElement);

    // Snapshot test - ensure button has no unsuspecting changes between commits
    const ElementTree = renderer
      .create(LoadingElement)
      .toJSON();

    expect(ElementTree).toMatchSnapshot();
  });
});
