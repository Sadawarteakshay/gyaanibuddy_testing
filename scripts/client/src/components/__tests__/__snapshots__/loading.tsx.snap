// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading render snapshot 1`] = `
<svg
  className="css-kahotv"
>
  logo.svg
</svg>
`;
